Sources/SnapshotTesting/Strategy/NSViewController.swift

```diff
@@ -7,6 +7,14 @@ extension Strategy where Snapshottable == NSViewController, Format == NSImage {
   }
 
   public static func image(precision: Float) -> Strategy {
+    return .image(precision: precision, size: nil)
+  }
+
+  public static func image(precision: Float = 1, size: CGSize) -> Strategy {
```

Comment:
Just some basic overloads for taking `precision`, `size`, or both, while delegating to a nullable, private implementation.